roles/container-engine/validate-container-engine/tasks/main.yml

```diff
@@ -151,3 +151,4 @@
       import_role:
         name: container-engine/cri-o
         tasks_from: reset
+        handlers_from: reset
```

Comment:
Could you make it more explicit than this a workaround to not import handlers ? Change the file name to something like 'none / not_existing /whatever'

Reply:
Any change you could address this ?